packages/website/docs/reverseSnippetHit.md

```diff
@@ -2,36 +2,84 @@
 id: reverseSnippetHit
 ---
 
-Returns a virtual node with non-matching parts of an Algolia hit snippet.
+Returns a virtual node with highlighted non-matching matching parts of an Algolia hit's snippet.
 
-## Example
+The `reverseSnippetHit` function lets you turn an Algolia hit's snippet into a virtual node with highlighted non-matching parts for a given attribute.
+
+## Examples
+
+### With a single string
+
+To determine what attribute to parse, you can pass it as a string.
 
 ```js
 import { reverseSnippetHit } from '@algolia/autocomplete-js';
 
-const hit = {}; // fetch an Algolia hit
+// An Algolia hit for query "hello"
+const hit = {
+  query: 'Hello there',
+  _snippetResult: {
+    query: {
+      value:
+        'Hello __aa-highlight__there__/aa-highlight__',
+    },
+  },
+};
 const reverseSnippetedValue = reverseSnippetHit({
   hit,
   attribute: 'query',
 });
 ```
 
-## Params
+### With nested attributes
+
+If you're referencing a nested attribute, you can use the array syntax.
+
+```js
+import { reverseSnippetHit } from '@algolia/autocomplete-js';
+
+// An Algolia hit for query "hello"
+const hit = {
+  query: {
```

Comment:
We can update the example for nested attributes like the other PRs.

Reply:
Addressed in 302c04df.